chapter1/exercise-1-39.rkt

```diff
@@ -0,0 +1,20 @@
+#lang planet neil/sicp
+
+; does not work
+
+(define (cont-frac n d k)
+  (define (iter i sum)
+    (if (= i 0)
+        sum
+        (iter (dec i) (/ (n k) (+ (d k) sum)))))
```

Comment:
I think the mix up is here.

The you want to use the the value you iterate with, not the initial parameter, i.e. `i` instead of `k`:

```racket
(iter (dec i) (/ (n i) (+ (d i) sum))
```

With that I'm getting these results:

```
1.557407724654902
0.5463024898437905
0.25534192122103627
0
```